### DISCUSSION TITLE
关于OCR识别和numlock数字键的相关问题反馈

### DISCUSSION THREAD

**Discussion by sumeco** [shuax/MouseInc.Settings#331] [Q&A]
你好，OCR识别失效，可否开放自定义API接口？或者更新下？
还有numlock数字键，可否设置成一定时间内自动取消数字锁定？

**Comment by gbsix** [comment]
是的，这两天发现OCR失效了：发生未知错误，请稍后在试

**Reply by Eunknight** [reply]
https://github.com/shuax/MouseInc.Settings/discussions/335#discussioncomment-3523123
